_data/compatibility/bitnob.yaml

```diff
@@ -0,0 +1,56 @@
+---
+name: Bitnob
+internal_url: /en/compatibility/bitnob
+logo: /img/compatibility/bitnob/bitnob.png
+rbf:
+  tested:
+    date: "2022-05-11"
+    platforms:
+      - Android
+    version: "10.0.0"
+  features:
+    receive:
+      notification: "na"
+      list: "na"
+      details: "na"
+      shows_replaced_version: "false"
+      shows_original_version: "false"
+    send:
+      signals_bip125: "false"
+      list: "untested"
+      details: "untested"
+      shows_replaced_version: "untested"
+      shows_original_version: "untested"
+  examples:
+    - image: /img/compatibility/bitnob/rbf/send-screen-default.png
+      caption: >
+        Sending Transaction - Default send transaction screen. Bitnob does not have option to enable or disable RBF when sending transactions.
+    - image: /img/compatibility/bitnob/rbf/send-fee-notice.png
+      caption: >
+        Sending Transaction - Network fee notice details.
+    - image: /img/compatibility/bitnob/rbf/transaction-list-incoming-rbf.png
+      caption: >
+        Receiving Transaction Signaling RBF - bitnob does not show unconfirmed transactions, only shows if a transaction is pending(that is, has not been confirmed on the blockchain)
```

Comment:
This sentence seems to be perhaps missing a word or is contradictory. Currently it says that Bitnob doesn't show unconfirmed transactions,  but it does show "pending" transactions, which it describes as not being confirmed on the blockchain. That's what an unconfirmed transaction is. Did you perhaps mean to say that "Bitnob doesn't show unconfirmed transactions _signaling RBF_", or something similar?

Reply:
Yeah exactly, what I meant. 
"Bitnob doesn't show unconfirmed transactions signaling RBF" sounds better.

Reply:
Would correct that.